examples/platforms/da15000/cpu.h

```diff
@@ -0,0 +1,52 @@
+/**
+ * CPU abstraction -- maps to raw register file.
```

Comment:
Is this file useful?  This file is specific to the DA15100.  This file is only included in 4 places.  I would prefer that we include only the necessary header files in each of those 4 locations.  It makes it more explicit what is actually needed by the given source files.